NOTICE.txt

```diff
@@ -0,0 +1,171 @@
+HDF5j: Java Implementation for HDF5
+Copyright 2017 Daniel Gomez-Sanchez
+
+This NOTICE.TXT contains notices of third-party software components that HDF5j is required to
+provide.
+
+This software contains code and files derived from the HDF group (https://www.hdfgroup.org/).
+Text from required COPYING files are included below.
+
+
```

Comment:
Add a separator from text and notices: something like:
`==== Third-party LICENSES/NOTICES ====`